src/main/java/com/github/manolo8/darkbot/Bot.java

```diff
@@ -18,6 +19,8 @@ public static void main(String[] args) {
             UIManager.setLookAndFeel(new FlatDarkLaf());
             UIManager.put("Button.arc", 0);
             UIManager.put("Component.arc", 0);
+
+            CommandLineUtils.parse(args);
```

Comment:
What does look & feel have to do with this line? Should be either before or after this try block, but not inside it